controllers/appgroup_reconciler.go

```diff
@@ -147,7 +147,8 @@ func (r *ApplicationGroupReconciler) reconcileApplications(l logr.Logger, appGro
 					}
 				}
 
-				scc.Metadata.Name = utils.ConvertToDNS1123(utils.ToInitials(appCh.Metadata.Name) + "-" + scc.Metadata.Name)
+				hashAppChName := utils.GetHash(appCh.Metadata.Name)
```

Comment:
Can we combine all these functions into a single `GetChartName` function? This will make things more readable